app/models/keyword.rb

```diff
@@ -12,6 +12,7 @@ def to_s
   end
 
   before_save do
+    self.term = self.term.gsub(Madek::Constants::TRIM_WHITESPACE_REGEXP, '')
```

Comment:
Unless there is some local variable with the same name overshadowing the reader method which is not the case here, so you are right. ;-)